src/inputcommand.js

```diff
@@ -35,6 +35,15 @@ export default class InputCommand extends Command {
 		 * @member {module:typing/utils/changebuffer~ChangeBuffer} #_buffer
 		 */
 		this._buffer = new ChangeBuffer( editor.model, undoStepSize );
+
+		/**
+		 * Stores batches created by the input command. The batches are used to differentiate input batches from other batches using
+		 * {@link module:typing/input~Input#isInput} method.
+		 *
+		 * @type {WeakSet<module:engine/model/batch~Batch>}
+		 * @protected
```

Comment:
Does it have to be `protected`? Why not `private`?

Reply:
It's used by other class (`Indent`) so this must be protected not private.

Reply:
Okay, I didn't notice these are in two different classes.